### DISCUSSION TITLE
Exporting video of labeled frames

### DISCUSSION THREAD

**Discussion by IsabelleSajonia** [talmolab/sleap#1521] [Help!]
I'd like to evaluate my model's performance in Python or matlab. I was wondering if it's possible to get a video output that shows all of my labeled frames on the mouse? I could screen record in the sleap GUI while moving through frames of my predictions but that would take longer, and the quality is not great. I'm interested in showing what the output looks like for this experiment in a casual presentation. Thank you!

**Comment by roomrys** [comment]
Hi @IsabelleSajonia,

You can render a video with instances overlayed by using View > Render Video Clip with Instances... or if you like the CLI, you can use the [`sleap-render`](https://sleap.ai/guides/cli.html#sleap-render) command.

Thanks,
Liezl
